src/uct/sm/mm/mm_iface.c

```diff
@@ -256,6 +258,37 @@ unsigned uct_mm_iface_progress(void *arg)
     return count;
 }
 
+static ucs_status_t uct_mm_iface_event_fd_get(uct_iface_h tl_iface, int *fd_p)
+{
+    *fd_p = ucs_derived_of(tl_iface, uct_mm_iface_t)->signal_fd;
+    return UCS_OK;
+}
+
+static ucs_status_t uct_mm_iface_event_fd_arm(uct_iface_h tl_iface,
+                                              unsigned events)
+{
+    uct_mm_iface_t *iface = ucs_derived_of(tl_iface, uct_mm_iface_t);
+    char dummy[32]; /* pop multiple signals at once */
```

Comment:
just some reasonable value of how many events to read at once
not to large to fit on the stack, and large enough to reduce overhead of reading events